CHANGELOG.md

```diff
@@ -1,6 +1,32 @@
 Changelog
 =========
 
+4.5
+---
+
+* Add support for OCaml 4.11.
+  - `Ppx_deriving.string_of_{constant,expression}_opt` to destruct
+    `Pconst_string` in a version-independent way
+  #220, #222
+  (kit-ty-kate, Thierry Martinez, reviewed by Gabriel Scherer)
+
+* Stronger type equalities in `Ppx_deriving_runtime` (for instance,
+  `Ppx_deriving_runtime.result` and `Result.result` are now compatible with
+  all OCaml versions)
+  #223, #225
+  (Thierry Martinez, reviewed by Gabriel Scherer)
+
+* `Ppx_deriving_runtime.Option` compatibility module
+  #222
+  (Thierry Martinez, reviewed by Gabriel Scherer)
+
+4.4.1
+-----
+
+* Add support for OCaml 4.10
+  #211
+  (kit-ty-kate, reviewed by Gabriel Scherer)
```

Comment:
@kit-ty-kate, should we rather use your github handle or a full name of your choice?

Reply:
full name is fine